### DISCUSSION TITLE
Wat te doen met fout (?) opgevoerde sensoren door andere bronhouder?

### DISCUSSION THREAD

**Discussion by CelineJansen** [kadaster-labs/sensrnet-home#303] [Q&A]
In https://viewer.sensorenregister.nl/ staan twee camera's binnen Eindhoven geregistreerd waar inderdaad een Citybeacon staat:
![image](https://user-images.githubusercontent.com/79267472/160766992-9682cddf-9a1c-4d5e-9333-822ce06d07f7.png)

![image](https://user-images.githubusercontent.com/79267472/160766775-2196e763-1fd9-470e-8960-855a21f215fa.png)

Voor zover ik (Eindhoven) weet zitten er geen werkende camera's aan deze beacon. Ik kan de sensoren ook niet verwijderen, want deze zijn opgevoerd door Gemeente Tilburg:
![image](https://user-images.githubusercontent.com/79267472/160767382-0fece91c-a8e7-480b-82e8-62a1ce3ed674.png)

Ik verwacht dat het hier om een foutje gaat dat bij het testen van de bulk opvoer erin geslopen is? Het gaat namelijk om de default omschrijving uit de opvoer sheet. 
Dit triggert bij mij de volgende vragen:

- Waar kunnen gebruikers dergelijke foutjes terugmelden zodat deze worden hersteld (heb ik zelf geen rechten toe)?
- Kan er een automatische foutafhandeling worden ingebouwd dat gemeentes geen sensoren binnen de grenzen van andere gemeentes kunnen plaatsen?
- Of kan worden ingebouwd dat gemeentes alle sensoren binnen hun bronhoudersgrenzen kunnen bewerken, ook als deze niet van hen zijn?

**Comment by marcvanandel** [comment]
😃 yup. Dit is typisch een 'probleem' van een decentrale opzet. Dit kan ... en we hebben als community hier nog te weinig ervaring opgedaan en ook niet besproken hoe we hier mee om willen gaan. Tegelijk is dit niet anders dan het beheer van de BGT, verwacht ik? Ben je dan als bronhouder cq gemeente beperkt tot je eigen gemeentelijke grenzen? Is dat hard in de software afgedwongen of ... ?

Voor dit moment kun je het beste contact opnemen met Tilburg (bijv. @mikegroels)

**Reply by CelineJansen** [reply]
Voor de BGT heb je je eigen interessegebied, maar daar binnen kan je wel codes van andere bronhouders gebruiken. Er zijn landelijke dashboards beschikbaar ter controle op zulke "zwerfobjecten" waarin gesignaleerd wordt wanneer een onverwachtste bronhouder optreedt in een gebied. En daar hebben we natuurlijk verbeterdekaart.nl om terug te melden.

**Reply by CelineJansen** [reply]
Maar voor nu: @mikegroels: kun jij in jullie registratie node iets met die twee citybeacons die namens Tilburg in Eindhoven geplaatst zijn? Als deze inderdaad per abuis door jullie zijn opgevoerd dan zou het super zijn wanneer je deze kan verwijderen. Bedankt alvast!

**Reply by mikegroels** [reply]
Waar die sensoren precies vandaan komen, geen idee, ik zou dat kunnen navragen. Zoals Marc al zegt, zal dit nog wel wat beter dichtgetimmerd moeten worden, met:
- Een terugmeldvoorziening, ongeacht wat de oorsprong van de fout is, naar de betreffende bronhouder (meestal gemeente)
- Een registratiebeperking. Voor een gemeente zou dit gelimiteerd kunnen worden op de gemeentegrenzen, maar ik neem aan dat dit technisch best wel een uitdaging kan zijn. Aan de andere kant wil je dat nutsbedrijven of andere landelijk opererende instanties dit soort limitaties niet ondervinden.

**Reply by mikegroels** [reply]
> Maar voor nu: @mikegroels: kun jij in jullie registratie node iets met die twee citybeacons die namens Tilburg in Eindhoven geplaatst zijn? Als deze inderdaad per abuis door jullie zijn opgevoerd dan zou het super zijn wanneer je deze kan verwijderen. Bedankt alvast!

Dat gaat even niet, de omgeving is uit de lucht door security issues m.b.t. Haven omgeving. Heb onze applicatiebeheerder gevraagd of dit inmiddels niet is opgelost. Ik houd je op de hoogte (of @marcvanandel / iemand@KAD moet de sensoren even verwijderen).

**Reply by CelineJansen** [reply]
Bedankt voor je snelle reactie, komt denk ik doordat dit de defaultwaardes waren in de bulkopvoer Excellijst... Als je die niet vervangt worden die ook opgevoerd telkens als je de bulkopvoer lijst gebruikt.

**Reply by mikegroels** [reply]
Klopt, was een foutje. Helaas is er nog geen zicht op een datum/moment waarop de boel weer in de lucht komt.

**Reply by marcvanandel** [reply]
Haha ... Wij kunnen niet even wat verwijderen, @mikegroels . Dat is nu juist het mooie van het systeem: Het is jullie data ... en daar kunnen wij niets aan veranderen. Zo zou het ook moeten zijn ... maar dat brengt wel met zich mee dat het jullie data blijft en dat jullie dat (dus) zelf goed moeten beheren 😉 

_Ieder voordeel heb z'n nadeel_ 😆

**Reply by mikegroels** [reply]
Sterk punt, dacht dat jullie jezelf toch wel een backdoor verschaft zouden hebben :)
@celine dan moet je even geduld hebben, hoop dat het niet te erg in de weg staat.

**Reply by CelineJansen** [reply]
Nee hoor Mike, geen probleem en geen haast. Blijft toch ook een beetje een pilot productje ;) Bedankt voor jullie reacties en de moeite.

**Reply by marcvanandel** [reply]
Nope. Geen backdoors. Never! 😃
